### ISSUE TITLE
TypeError: item.destroy is not a function

### ISSUE THREAD

**Issue by citricacid-pl** [hilongjw/vue-lazyload#380]
Hi, I am using vue-lazyload version: 1.3.1 and vue version: 2.6.10.

I am receiving TypeError while using `<lazy-component>`:

```
Uncaught TypeError: item.destroy is not a function
    at eval (vue-lazyload.esm.js?0177:1349)
    at Array.forEach (<anonymous>)
    at Lazy._lazyLoadHandler (vue-lazyload.esm.js?0177:1347)
    at runCallback (vue-lazyload.esm.js?0177:484)
```

![Zrzut ekranu 2019-07-11 o 10 35 49](https://user-images.githubusercontent.com/745572/61035814-215c6200-a3c8-11e9-9cda-cbd0335dea81.png)

**Comment by coleShang** [comment]
me too

**Comment by stefanfisk** [comment]
A quick and DIRTY workaround is to do this before loading the plugin:

**Comment by citricacid-pl** [comment]
@stefanfisk Yes. Seems dirty. There is another solution proposed in #381 (similar to yours but not that hacky)

**Comment by AmeerTaweel** [comment]
> A quick and DIRTY workaround is to do this before loading the plugin:
> 
> ```
> Vue.use({
>   install() {
>     Vue.prototype.destroy = Vue.prototype.$destroy;
>   },
> });
> ```

Thanks. This worked for me.

**Comment by JensvdHeydt** [comment]
@hilongjw  The fix to this problem is already merged into master. Could you please make a new release, so that all these issues can get resolved?
